manager.py

```diff
@@ -183,3 +194,29 @@ def get_log_entries(
                     files.append(path.join(logs_dir, filename))
 
             return LogEntries.read_all(files, entries_count, level, component)
+
+    def _identify_service(self, service):
```

Comment:
Maybe this should be a function of the service manager? I can see this being useful elsewhere and I think it already is in use in the core